src/test/resources/discovery/specification-sample-1.html

```diff
@@ -11,17 +11,20 @@ <h1>Contents</h1>
     </ul>
 </section>
 <section about="https://example.org/specification1#spec1" typeof="spec:NormativeRequirement">
-    <h1>Requirement 1</h1>
+    <h1 property="schema:name">Requirement 1</h1>
+    <p property="schema:description">Full <span property="spec:excerpt">text of requirement 1</span></p>
     <dl><dt>Role</dt><dd property="spec:conformanceRole" resource="spec:ServerConformanceClass">Server</dd></dl>
     <dl><dt>Level</dt><dd property="spec:normativeConformanceLevel" resource="spec:ConformanceMust">MUST</dd></dl>
 </section>
 <section about="https://example.org/specification1#spec2" typeof="spec:NormativeRequirement">
-    <h1>Requirement 2</h1>
+    <h1 property="schema:name">Requirement 2</h1>
+    <p property="schema:description">Full <span property="spec:excerpt">text of requirement 2</span></p>
     <dl><dt>Role</dt><dd property="spec:conformanceRole" resource="spec:ServerConformanceClass">Server</dd></dl>
     <dl><dt>Level</dt><dd property="spec:normativeConformanceLevel" resource="spec:ConformanceMust">MUST</dd></dl>
```

Comment:
Reminds me to update the spec vocab so that these are either namedindividuals or used as instances.